server/lib/twitter/twitter_feed_stream.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Module streams twitter feed
+ **/
+
+import AbstractFeedStream from './../abstract_feed_stream';
+import Q from 'q';
+import Twit from 'twit';
+import { chain, extend } from 'underscore';
+import config from '../../config/config';
+
+export default class TwitterFeedStream extends AbstractFeedStream {
+
+  constructor() {
+    super();
+    this.twitterApi = new Twit({
+      consumer_key: config.TWITTER_CONSUMER_KEY,
+      consumer_secret: config.TWITTER_CONSUMER_SECRET,
+      access_token: config.TWITTER_ACCESS_TOKEN,
+      access_token_secret: config.TWITTER_ACCESS_TOKEN_SECRET,
+    });
+  }
+
+  _buildParametersForNextPageRequest(originalReqParams, lastDataResponse) {
+    const maxId = chain(lastDataResponse).pluck('id').min().value();
```

Comment:
Yeah, it may not be obvious when you look at it for the first time :smiley_cat: I put a piece of Twitter spec in the documentation of this method. Added in 0e5c541d. Thanks!